src/EnergyPlus/DXCoils.cc

```diff
@@ -7963,7 +7963,7 @@ void SizeDXCoil(EnergyPlusData &state, int const DXCoilNum)
             CompName = thisDXCoil.Name;
             CompType = thisDXCoil.DXCoilType;
             if (Mode == thisDXCoil.NumOfSpeeds) {
-                FieldNum = 10 + (Mode - 1) * 13;
+                FieldNum = 10 + (Mode - 1) * 14;
```

Comment:
A new input was recently added so a change in the increment needed to be changed.